pkgs/tools/system/auto-epp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "auto-epp";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "jothi-prasath";
+    repo = "auto-epp";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7sI8K+7ZAdzBN/XOzYQQZ1f9t+fFo6fcXYzX6abNyQ8=";
+  };
+
+  buildInputs = [ python3 ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 auto-epp $out/bin/auto-epp
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    mainProgram = pname;
```

Comment:
nit: while i understand the intent here, it's semantically wrong. The executable doesn't necessarily have the same name as the package - it just happens to be the case here (and for many other packages as well).

Reply:
sure, i'll hardcode it